views/MapsContainer.tsx

```diff
@@ -12,7 +13,11 @@ class MapsContainer extends React.PureComponent<MapsContainerProps> {
             <Maps
                 gpsLocation={this.props.gps.data.at(-1) || {}}
                 gpsPath={this.props.gps.data.map(
-                    (gpsCoordinate: GPSCoordinate) => convertToLatLng(gpsCoordinate)
+                    (gpsCoordinates: GPS) => convertToLatLng(gpsCoordinates)
```

Comment:
Thanks for catching the type mismatch and changing this to `GPS`! 

Could we rename `gpsCoordinates` to `gpsPoint`?